ui/page/wallets/privacy_page.go

```diff
@@ -426,7 +426,18 @@ func (pg *PrivacyPage) showModalSetupMixerAcct() {
 func (pg *PrivacyPage) showModalPasswordStartAccountMixer() {
 	modal.NewPasswordModal(pg.Load).
 		Title("Confirm to mix account").
-		NegativeButton("Cancel", func() {}).
+		NegativeButton("Cancel", func() {
+			info := modal.NewInfoModal(pg.Load).
+				Title("Cancel mixer?").
+				Body("Are you sure you want to cancel mixer action?").
+				NegativeButton("No", func() {
+					pg.showModalPasswordStartAccountMixer()
+				}).
+				PositiveButton("Yes", func() {
+					pg.toggleMixer.SetChecked(false)
+				})
+			pg.ShowModal(info)
+		}).
```

Comment:
```suggestion
		NegativeButton("Cancel", func() {
			pg.toggleMixer.SetChecked(false)
		}).
```
this is all you need as the mixing process has not started.

To set exit the info modal, you will need to add it here https://github.com/planetdecred/godcr/blob/977d09b6c16b4bd758b751c1e596e37aa0e4115f/ui/page/privacy_page.go#L347.